<launch>
  <!-- Run the map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find robbie)/maps/3_jan16_2014.yaml"/>

  <!--- Run AMCL -->
  <include file="$(find robbie)/launch/amcl_diff.launch" />

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find robbie)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find robbie)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find robbie)/params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find robbie)/params/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find robbie)/params/base_local_planner_params.yaml" command="load" />
  </node>
</launch>
